test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NetworkCallCountTest.cs

```diff
@@ -1197,7 +1197,13 @@ public void NetworkCallCount_RestoreSolutionMultipleSourcesTwoV3()
                     slnPath,
                     "-Verbosity",
                     "detailed",
-                    "-DisableParallelProcessing"
+                    "-DisableParallelProcessing",
+                    "MSBuildVersion",
```

Comment:
Found 16 other occurrences of `msbuildversion` in tests. Should we also cover those scenarios for Dev15?